src/test.ts

```diff
@@ -9,24 +9,23 @@ import 'zone.js/dist/fake-async-test';
 import { getTestBed } from '@angular/core/testing';
 import {
   BrowserDynamicTestingModule,
-  platformBrowserDynamicTesting
+  platformBrowserDynamicTesting,
 } from '@angular/platform-browser-dynamic/testing';
 
 // Unfortunately there's no typing for the `__karma__` variable. Just declare it as any.
-declare var __karma__: any;
-declare var require: any;
+declare const __karma__: any;
+declare const require: any;
 
 // Prevent Karma from running prematurely.
-__karma__.loaded = function () {};
+__karma__.loaded = function() {};
 
 // First, initialize the Angular testing environment.
 getTestBed().initTestEnvironment(
   BrowserDynamicTestingModule,
   platformBrowserDynamicTesting()
 );
 // Then we find all the tests.
-// const context = require.context('./', true, /\.spec\.ts$/);
-const context = require.context('./', true, /specs\.ts$/);
+const context = require.context('./', true, /\.spec\.ts$/);
```

Comment:
Try to return to the spec format. The idea is to use prural to sort the tests by features

Reply:
fizz-buzz.spec.ts -> specs.ts
main.spec.ts -> specs.ts